src/shipchain_common/iot.py

```diff
@@ -14,31 +14,20 @@
 limitations under the License.
 """
 
-import json
 import logging
 import re
 
 import requests
 from aws_requests_auth.boto_utils import BotoAWSRequestsAuth
 from django.conf import settings
-from rest_framework import status
-from influxdb_metrics.loader import log_metric, TimingMetric
-
-from .exceptions import AWSIoTError
-from .utils import DecimalEncoder
 
+from .aws import AWSClient
 
 LOG = logging.getLogger('python-common')
+# pylint: disable=too-many-arguments
```

Comment:
?? What's this for?

Reply:
That was there originally as I increased the amount of arguments for the call by 1 and it caused a pylint issue, but I can remove it now.